### ISSUE TITLE
field sorting 

### ISSUE THREAD

**Issue by mairaw** [MicrosoftDocs/feedback#351]
Original GitHub issue https://github.com/dotnet/docs/issues/5218 by @mefasoft

It will be better if field section can be sorted by enum value.

* Content: [TypeCode Enum (System)](https://docs.microsoft.com/en-us/dotnet/api/system.typecode?view=netframework-4.7.2)

**Comment by mairaw** [comment]
I'm wondering if we even should display that column at all. Thoughts @rpetrusha @BillWagner?

**Comment by rpetrusha** [comment]
MSDN didn't display underlying enumeration values, and our previous approach to documenting enumeration members was to manually document enumeration values only if there was a compelling reason to do so (like you'd want to pass the value as an argument to some other method). In most cases, the enumeration member name is intended to be an abstraction that allows you to ignore the underlying value.
So to put it another way, I'd prefer sorting by field name as well, and suppress the column that lists the enumeration member's underlying value.
//cc @mairaw @BillWagner

**Comment by BillWagner** [comment]
I spent some time thinking about when I needed or wanted the enum values in a code base.  It came down to a couple scenarios:

1. When the Enum is a bit field, and I need to determine values in the debugger.
1. When an enum is cast to or from a numeric value and the semantic info about the enum was lost (databases, REST packets, and so on)

Neither of those are mainline scenarios, so I agree with @mairaw and @rpetrusha

**Comment by dend** [comment]
Chiming in here - in the spirit of having comprehensive documentation, that's something that was previously asked by the Windows team and by customers. Given that there are still useful scenarios that the value might be relevant for, I suggest for the time being we `wont-fix` this and keep the current alpha ordering, as well as the values.

**Comment by dend** [comment]
@mairaw @BillWagner can you please confirm whether the plan stated above is OK?

**Comment by mairaw** [comment]
I think you had enough feedback from the team @dend on what would be our ultimate preference.
Given the choice of keeping the current sorting or changing it to the field value, I'd keep as-is and mark this as wont-fix.

**Comment by mairaw** [comment]
We seem to be getting a lot of PRs to change this behavior on API docs. We should re-examine this.

**Comment by xtqqczze** [comment]
> We seem to be getting a lot of PRs to change this behavior on API docs. We should re-examine this.

Indeed: https://github.com/dotnet/dotnet-api-docs/pull/8892, https://github.com/dotnet/dotnet-api-docs/pull/6786, https://github.com/dotnet/dotnet-api-docs/pull/6749, https://github.com/dotnet/dotnet-api-docs/pull/4725, https://github.com/dotnet/dotnet-api-docs/pull/4558, https://github.com/dotnet/dotnet-api-docs/pull/4352, https://github.com/dotnet/dotnet-api-docs/pull/3852.

**Comment by gewarren** [comment]
This is fixed now - enum fields are sorted by value.
